skynet.py

```diff
@@ -6,13 +6,16 @@
 commit_hash = os.getenv("SKYNET_BUILD_COMMIT_HASH")
 branch_name = os.getenv("SKYNET_CHECKOUT_REF").split("/")[-1]
 github_token = os.getenv("GITHUB_TOKEN")
+query = {"branch": branch_name}
+if "." in branch_name:
+    query = {"actor": "rmconsole-readonly-wk"}
```

Comment:
I'm not sure I'm following - what does this do?

Reply:
Before we were querying for the github actions runs by branch name but the branch name on a release branch is  something like 4.0.5 which means the URL that is being hit to get the run looks like this:  `https://api.github.com/repos/Workiva/w_transport/actions/runs?branch=4.0.5` which doesn't work because of the dots in it. The release branches are all made by rosie with the username `rmconsole-readonly-wk` so this is just looking for runs that were triggered by that user instead of by branch name.